bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -106,7 +106,7 @@ public interface VoiceManager {
      * Interprets the passed string using a particular HLI service and the default locale.
      *
      * @param text The text to interpret
-     * @param hliId The id of the HLI service to use or null
+     * @param hliId Comma separated list of HLI service ids to use or null
```

Comment:
Please rename the parameter into `hliIdList` (name you are using in your implementation and looks good to me)